grakn-engine/src/main/java/ai/grakn/engine/controller/SystemController.java

```diff
@@ -71,10 +76,45 @@
     private final Logger LOG = LoggerFactory.getLogger(SystemController.class);
     private final EngineGraknGraphFactory factory;
 
-    public SystemController(EngineGraknGraphFactory factory, Service spark) {
+    public SystemController(EngineGraknGraphFactory factory , Service spark) {
         this.factory = factory;
-        spark.get(KEYSPACES,     this::getKeyspaces);
-        spark.get(CONFIGURATION, this::getConfiguration);
+
+        spark.get(INITIALISE,         this::initialiseSession);
```

Comment:
I don't see those as valid enough use cases to maintain two controllers which essentially do the same thing. Just more code to maintain in my eyes. I would make the keyspace parameter optional and just have one endpoint.